Auto-Configuration does not work with Spring Boot 3

Auto-configuration must be done via `META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports` now.
> Rather than a single comma-separate list, each line contains the fully qualified name of an auto-configuration class.

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.7-Release-Notes#changes-to-auto-configuration
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0-Migration-Guide#auto-configuration-files

If I import the configuration manually, I still get errors in Spring Boot 3 though:
```
Caused by: java.lang.NoSuchMethodError: 'void org.springframework.web.server.ResponseStatusException.<init>(org.springframework.http.HttpStatus, java.lang.String)'
	at com.github.JanLoebel.jsonschemavalidation.JsonSchemaValidationException.<init>(JsonSchemaValidationException.java:15)
	at com.github.JanLoebel.jsonschemavalidation.provider.DefaultJsonSchemaProvider.handleValidationMessages(DefaultJsonSchemaProvider.java:31
...
```
I think that we might need to recompile against Spring 6.

--- a/pom.xml
+++ b/pom.xml
@@ -10,7 +10,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>8</java.version>
-        <spring-boot.version>2.4.3</spring-boot.version>
+        <spring-boot.version>3.0.0</spring-boot.version>
         <json-schema-validator.version>1.0.49</json-schema-validator.version>
     </properties>
 

--- a/src/main/resources/META-INF/spring.factories
+++ b/src/main/resources/META-INF/spring.factories
@@ -1,1 +1,0 @@
-org.springframework.boot.autoconfigure.EnableAutoConfiguration=com.github.JanLoebel.jsonschemavalidation.JsonSchemaValidationAutoConfiguration

--- a/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports
+++ b/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports
@@ -1,0 +1,1 @@
+com.github.JanLoebel.jsonschemavalidation.JsonSchemaValidationAutoConfiguration

